assets/js/data/shared-controls.js

```diff
@@ -55,6 +55,7 @@ export const controls = {
 						} );
 				} )
 				.catch( ( errorResponse ) => {
+					triggerFetch.setNonce( errorResponse.headers );
```

Comment:
We already do this in Checkout processor, but this is not inside that context, so it doesn't happen.